keyboards/unison/v04/config.h

```diff
@@ -0,0 +1,125 @@
+/*
+Copyright 2021 Takeshi Nishio
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID 0x2403
+#define PRODUCT_ID 0x176A
+#define DEVICE_VER 0x0040
+#define MANUFACTURER jpskenn
+#define PRODUCT Unison
+
+/* key matrix size */
+/* NOTE: With Round-Robin matrix, set same size for both. */
+#define MATRIX_ROWS 10
+#define MATRIX_COLS 10
+
+/* key matrix pins */
+/* NOTE: With Round-Robin matrix, set same pins for both. */
+#define MATRIX_ROW_PINS { B3, E6, F1, F5, F7, B2, F0, F4, F6, C7 }
+#define MATRIX_COL_PINS { B3, E6, F1, F5, F7, B2, F0, F4, F6, C7 }
+#define UNUSED_PINS
+
+#define DIODE_DIRECTION COL2ROW
+
+/* Rotary Encoder */
+#ifdef ENCODER_ENABLE
+  #define ENCODERS_PAD_A { B0, D2, D5, D6, B4 }
+  #define ENCODERS_PAD_B { B1, D3, D4, D7, B5 }
+  #define ENCODER_RESOLUTION 4  //the default & suggested is 4
+#endif
+
+/* Audio */
+#ifdef AUDIO_ENABLE
+  #define B6_AUDIO
+  #define C6_AUDIO
```

Comment:
```suggestion
  #define B6_AUDIO
  #define C6_AUDIO
#    define AUDIO_PIN C6
#    define AUDIO_PIN_ALT B6
```

Reply:
What is the settings about `AUDIO_PIN` and `AUDIO_PIN_ALT`?
I couldn't find that on QMK docs.

What I found on google is these.
https://github.com/qmk/qmk_firmware/issues/8825
https://git.skj.dev/sean/qmk_firmware/commit/ad5cbe59820ac1f9922f2db2d6d1e7f2a8f50519

It looks the settings is suit for STM32.
Do I have to apply these for my board with ATmega32u4?

Reply:
Mostly, it's preemptive, as there is a significant change in `develop` that will require any new boards to use these settings. 

If you want, this can be ignored, and I'll take care of it before develop is merged

Reply:
Thanks to inform about that.
I decided to include it for incoming change.

Reply:
been merged into master now, so: 
```suggestion
  #define AUDIO_PIN C6
  #define AUDIO_PIN_ALT B6
```